target/linux/ramips/image/rt305x.mk

```diff
@@ -1086,6 +1086,20 @@ define Device/unbranded_a5-v11
 endef
 TARGET_DEVICES += unbranded_a5-v11
 
+define Device/tenda_a6
+  SOC := rt5350
+  IMAGE_SIZE := 3776k
+  IMAGES += factory.bin
+  IMAGE/factory.bin := $$(sysupgrade_bin) | check-size | \
+	poray-header -B A5-V11 -F 4M
+  DEVICE_VENDOR := Tenda
+  DEVICE_MODEL := A6
+  DEVICE_PACKAGES := kmod-usb-ohci kmod-usb2
+  SUPPORTED_DEVICES += a6
+  DEFAULT := n
+endef
+TARGET_DEVICES += unbranded_a5-v11
```

Comment:
Copy-paste error. So, did you even test it?

Reply:
Everything was working on tarball-based build, I had to redo it for the github master branch in order to submit the patch. I quickly posted a follow-up commit to address this error, but yes as you have noted there are some further inconsistencies like the missing entry in 02_network. As far as marking 0x0 in the DTS file, I feel inclined to argue that you are being overly pedantic, as I used the accepted `unbranded_a5-v11` as base template, which according to you would be erroneous as well. You may want to check that out :) Regards

Reply:
The 0x0 is entirely cosmetic, it will work as well as just "0", but I prefer the former as it will match the other locations which are typically in hex notation. However, this is the least important of all request IIRC.
Despite, if the improvements we could ask for would always be limited to what somebody accepted for some devices in the past, there would never be improvement. I give feedback based on what I think would be best for supporting the particular device at hand.

Reply:
You are right. Look, I concede that this is mostly a copy-paste operation for the most part, but it's what was largely the basis of what was required to get this device running OpenWrt at all, if that makes sense. Appreciate all of your time and will try be more careful and compliant for the next pull, if I find the time to make the requested adjustments, both of the necessary and cosmetic kinds. Take care :D